docs/UserMenu.md

```diff
@@ -0,0 +1,107 @@
+</div>
+
+```typescript
+// app.module.ts
+import { UserMenuModule, InfoListItemModule } from '@pxblue/angular-components';
+...
+imports: [
+    UserMenuModule,
+    InfoListItemModule
+],
+...
+```
+
+```tsx
+const items = [
+    {
+        title: 'Account',
+        icon: 'settings',
+    },
+    {
+        title: 'Contact Us',
+        icon: 'mail',
+    },
+    {
+        title: 'Log Out',
+        icon: 'logout',
+    }
+];
+```
+
+```html
+<pxb-user-menu avatarValue="HA" menuTitle='Sample Title '[(open)]="state.open">
+    <mat-nav-list pxb-body [style.paddingTop.px]="0">
+        <pxb-info-list-item *ngFor="let item of items" [dense]="true">
+            <mat-icon pxb-icon>{{item.icon}}</mat-icon>
+            <div pxb-title>{{item.title}}</div>
+        </pxb-info-list-item>
+    </mat-nav-list>
+</pxb-user-menu> 
+```
+
+## API
+
+Parent element (`<pxb-user-menu>`) attributes:
+
+<div style="overflow: auto;">
+
+| @Input       | Description                                      | Type                    | Required | Default |
+| ------------ | ------------------------------------------------ | ----------------------- | -------- | ------- |
+| avatarImage  | Image source for avatar                          | `string`                | no       |         |
+| avatarValue  | Text value for avatar                            | `string`                | no       |         |
+| menuSubtitle | Subtitle shown when menu is open                 | `string`                | no       |         |
+| menuTitle    | Title shown when menu is open                    | `string`                | no       |         |
+
+</div>
+
+<div style="overflow: auto;">
+
+| @Output        | Description                                      | Type                    |
+| -------------- | ------------------------------------------------ | ----------------------- | 
+| backdropClick  | Emits event when backdrop is clicked             | `EventEmitter<void>`    | 
+| openChange     | Emits an event when the open prop changes        | `EventEmitter<boolean>` | 
+| select         | Emits the title of the selected menu item        | `EventEmitter<string>`  |
+
+</div>
+
+The following child elements are projected into `<pxb-user-menu>`:
+
+<div style="overflow: auto;">
+
+| Selector           | Description                                                                    | Required | 
+| ------------------ | ------------------------------------------------------------------------------ | -------- | 
+| [pxb-avatar]       | Custom avatar to show, alternative to `avatarImage` or `avatarValue` prop  | no       | 
+| [pxb-menu-avatar]  | Custom menu avatar to show                                                     | no       |
+| [pxb-header]       | Custom menu header content                                                     | no       |
+| [pxb-body]         | Custom menu body content                                                       | no       | 
```

Comment:
do we use pxb-body elsewhere? would pxb-menu-body be more descriptive?

Reply:
I updated pxb-body and pxb-header to be pxb-menu-header and pxb-menu-body.